src/t_list.c

```diff
@@ -905,7 +890,7 @@ void blockingPopGenericCommand(client *c, int where) {
     }
 
     /* If the list is empty or the key does not exists we must block */
-    blockForKeys(c,BLOCKED_LIST,c->argv + 1,c->argc - 2,timeout,NULL,NULL);
+    blockForKeys(c,BLOCKED_LIST,c->argv + 1,c->argc - 2,timeout,NULL,where,0,NULL);
```

Comment:
I thought about passing -1 since it's a signed int, maybe we could define LIST_NONE as -1?

The struct seems like a good option too, I'll experiment with that.

Reply:
i suppose -1 is ok too